docs/development.rst

```diff
@@ -213,6 +230,103 @@ active, as long as the Python `tox` package is installed in it:
     $ tox -e py27                      # Run tests on Python 2.7
 
 
+.. _`Testing from the source archives on Pypi or GitHub`:
+
+Testing from the source archives on Pypi or GitHub
+--------------------------------------------------
+
+The wheel distribution archives on Pypi
+(e.g. ``pywbem-1.0.0-py2.py3-none-any.whl``)
+contain only the files needed to run pywbem, but not the files needed to test
+it.
+
+The source distribution archives on Pypi and GitHub
+(e.g. ``pywbem-1.0.0.tar.gz``)
+contain all files that are needed to test pywbem.
```

Comment:
Should this be "needed to run and to test pywbem"  or "the files to test pywbem in addition to the files to run it"

Reply:
DONE